client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,381 @@
+	},
+	{
+		title: translate( 'Edit my menu' ),
+		link: `/customize/${ siteSlug }`,
+		synonyms: [ 'menu' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( 'Set a site logo' ),
+		link: `/customize/${ siteSlug }`,
+		synonyms: [ 'logo', 'identity' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( 'Find a plan to suit my site' ),
+		link: `/plans/${ siteSlug }`,
+		synonyms: [ 'upgrade', 'business', 'professional', 'personal' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'Cancel my plan' ),
+		link: `/me/purchases/${ siteSlug }`,
+		synonyms: [ 'upgrade', 'business', 'professional', 'personal' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'Upgrade my plan' ),
+		link: `/plans/${ siteSlug }`,
+		synonyms: [ 'upgrade', 'business', 'professional', 'personal' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'Cancel G Suite' ),
+		link: `/me/purchases/${ siteSlug }`,
+		synonyms: [ 'upgrade', 'business', 'professional', 'personal', 'google' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'Renew my plan' ),
+		link: `/plans/${ siteSlug }`,
+		synonyms: [ 'upgrade', 'business', 'professional', 'personal', 'plan' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'Renew my domain' ),
+		link: `/plans/${ siteSlug }`,
+		synonyms: [ 'domain', 'business', 'professional', 'personal', 'plan' ],
+		icon: 'plans',
+	},
+	{
+		title: translate( 'View my site activity' ),
+		link: `/activity-log/${ siteSlug }`,
+		icon: 'history',
+	},
+	{
+		title: translate( "View my site's latest stats" ),
+		link: `/stats/day/${ siteSlug }`,
+		synonyms: [ 'analytics' ],
+		icon: 'stats-alt',
+	},
+	{
+		title: translate( 'Upload an image, video, audio or document' ),
+		link: `/media/${ siteSlug }`,
+		synonyms: [ 'media', 'photo' ],
+		icon: 'image',
+	},
+	{
+		title: translate( 'Import content from another site' ),
+		link: `/settings/import/${ siteSlug }`,
+		synonyms: [ 'medium', 'blogger', 'wix', 'squarespace' ],
+		icon: 'cloud-upload',
+	},
+	{
+		title: translate( 'Earn money from my site' ),
+		description: translate(
+			"By upgrading to the Premium plan, you'll be able to monetize your site through the WordAds program."
+		),
+		link: `/earn/${ siteSlug }`,
+		synonyms: [ 'monetize', 'wordads', 'premium' ],
+		icon: 'money',
+	},
+	{
+		title: translate( 'Learn how to market my site' ),
+		link: `/marketing/tools/${ siteSlug }`,
+		synonyms: [ 'marketing', 'brand', 'logo', 'seo', 'tools', 'traffic' ],
+		icon: 'speaker',
+	},
+	{
+		title: translate( "Manage my site's users" ),
+		description: translate( 'Invite new users and edit existing ones.' ),
+		link: `/people/team/${ siteSlug }`,
+		synonyms: [ 'administrator', 'editor', 'contributor', 'viewer', 'follower' ],
+		icon: 'user',
+	},
+	{
+		title: translate( 'Invite new users to my site' ),
+		link: `/people/new/${ siteSlug }`,
+		synonyms: [ 'administrator', 'editor', 'contributor', 'viewer', 'follower' ],
+		icon: 'user',
+	},
+	{
+		title: translate( "Change my site's timezone" ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Change your site's address' )` *ES Score: 6*